ports/mimxrt/Makefile

```diff
@@ -322,14 +357,6 @@ else
     SRC_HAL_C += hal/$(FLEXSPI_FLASH_TYPE)_config.c
 endif
 
-ifeq ($(MICROPY_PY_BLUETOOTH),1)
```

Comment:
Unless there was a reason to move these two bits up in the file, I suggest keeping them where they were (ie not removing them from here and not adding them above).